File: Java/irrgarten/irrgarten.java
package irrgarten;

import irrgarten.UI.GraphicUI;
import irrgarten.controller.Controller;

public class irrgarten {
    static public void main(String[] args) {
        
        Game game = new Game(2);
        GraphicUI view = new GraphicUI();

        Controller controller = new Controller(game, view);
        controller.play();
    }
}

File: Ruby/Irrgarten/irrgarten.rb
require_relative 'controller/controller'
require_relative 'UI/textUI'
require_relative 'game'

module Irrgarten
  class Main
    def self.run
      controller = Control::Controller.new(Game.new(2), UI::TextUI.new)
      controller.play()
    end
  end
end

Irrgarten::Main.run